### PR TITLE
Added support for memory.events stats

### LINKED ISSUE #149: cgroup2: support statting memory.events*
The PR for `docker stats` (https://github.com/moby/moby/pull/40657) require `containerd/cgroups` support for `memory.events*`.

**EDIT**: What we can add now seems only `FailCnt`, and this is not urgent (not used by `docker` CLI)

### PR DESCRIPTION
**Pull request by Zyqsempai**
Fixes #149 
Added support for memory.events stats.

Signed-off-by: Boris Popovschi <zyqsempai@mail.ru>

### CODE REVIEW

File: v2/stats/metrics.proto [hunk 1]
```diff
@@ -63,6 +63,7 @@ message MemoryStat {
 	uint64 usage_limit = 33;
 	uint64 swap_usage = 34;
 	uint64 swap_limit = 35;
+	uint64 fail_cnt = 36;
```
Comment (AkihiroSuda) [inline v2/stats/metrics.proto]: field name should correspond to memory.events file

Comment (Zyqsempai) [reply v2/stats/metrics.proto]: Not sure that, `max` will be explanatory enough.

Comment (AkihiroSuda) [reply v2/stats/metrics.proto]: events_max?

**Comment by Zyqsempai** [comment]
@AkihiroSuda Do you have any idea how we can test memory events in UT?

**Comment by AkihiroSuda** [comment]
I think you can just add UT for `readKVStatsFile()`.

Also please provide the logs that shows how you tested this PR with cgctl CLI.

**Comment by AkihiroSuda** [comment]
ping @Zyqsempai

**Comment by Zyqsempai** [comment]
@AkihiroSuda Sorry, tomorrow will try to finish it up!

File: v2/manager.go [hunk 2]
```diff
@@ -426,6 +432,7 @@ func (c *Manager) Stat() (*stats.Metrics, error) {
 		UsageLimit:            getStatFileContentUint64(filepath.Join(c.path, "memory.max")),
 		SwapUsage:             getStatFileContentUint64(filepath.Join(c.path, "memory.swap.current")),
 		SwapLimit:             getStatFileContentUint64(filepath.Join(c.path, "memory.swap.max")),
+		EventsMax:             getUint64Value("max", memoryEvents),
```
Comment (AkihiroSuda) [inline v2/manager.go:435]: Can we also add other fields?

**Comment by AkihiroSuda** [comment]
Also, please squash commits

File: v2/manager.go (hunk 2 above)
Comment (Zyqsempai) [reply v2/manager.go:435]: Do we need them? I mean, they used somewhere?

Comment (AkihiroSuda) [reply v2/manager.go:435]: Not now probably, bu

Comment (Zyqsempai) [reply v2/manager.go:435]: Got it.Ok.

**Comment by AkihiroSuda** [comment]
ping @Zyqsempai

**Comment by AkihiroSuda** [comment]
carried as https://github.com/containerd/cgroups/pull/160

**Pull request closed without merging**